src/editor.js

```diff
@@ -1907,62 +1501,36 @@ export class OdooEditor extends EventTarget {
             'unordered': 'UL',
             'checklist': 'CL',
         };
-        this._protect(() => {
```

Comment:
Il est parti où le this._protect ?

Reply:
Il se trouve dans le `execCommand` > `_applyCommand`.
Avant mon changement, il y avait des commandes qui ne passait pas par là. Maintenant elles passent toute par `execCommand`.